wolfcrypt/src/random.c

```diff
@@ -2060,6 +2060,24 @@ int wc_GenerateSeed(OS_Seed* os, byte* output, word32 sz)
         return 0;
     }
 
+#elif defined(WOLFSSL_ESPIDF)
+    #if defined(WOLFSSL_ESPWROOM32)
+        #include <esp_system.h>
+        
+        int wc_GenerateSeed(OS_Seed* os, byte* output, word32 sz)
+        {
+            int i;
+            
+            for (i = 0; i< sz; i++) {
+               output[i] =  esp_random( );
+            }
+        
+            return 0;
+        }
+    #else
+        #define USE_TEST_GENSEED
```

Comment:
Let's remove this.  We want the user's compiler to give them a warning if they need to write their own GenerateSeed() which we do not support yet.  Having this here could make it easier for people to ship without a valid GenerateSeed() implemented.

Reply:
OK. I am going to remove this.

Reply:
Thanks